### DISCUSSION TITLE
Update customise columns on entries view to per user / group

### DISCUSSION THREAD

**Discussion by LauraMontgomery** [craftcms/cms#11532] [Ideas] [answered]
I agree with the update in v4 that columns in the entries view are now part of project config and can only be edited in development mode; therefore preventing users from overwriting the entries view layout.

It would be nice to be able to do this on a per user (or probably better, per user group) level, so that each user / group could have a customised experience depending on the information they might like to be able to see at a glance. I would still see this as part of project config, but something that could be unique per user / group - much like other user settings for sections, categories, etc.

**Comment by brandonkelly** [comment]
I’m not sure how we’d do that since users can belong to multiple groups. Each individual table attribute would probably need its own “User Groups” checkbox list, determining which groups are allowed to see the column.

We are considering allowing each user to customize the visible table columns, so the project config is only storing the _default_ list. (#11355)

**Reply by LauraMontgomery** [reply]
That's a good point about the multiple user groups. But yes, the ability for each user to customize the columns would be great. Thanks for considering it :)

**Comment by brandonkelly** [comment, marked as answer]
We’ve resolved this for Craft 4.3:

- Element indexes no longer show table columns that are conditionally hidden for the logged-in user (#11913)
- Users now have the ability to customize which columns are visible for themselves (#11915).
